### PR TITLE
[CLDNN] Add unittests for Permute optimized kernel

### PR DESCRIPTION
**Pull request by EunsooSheenIntel**

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp [hunk 1]
```diff
@@ -6047,6 +6047,21 @@ struct permute_params {
 #define CASE_PERMUTE_U8_3D_2 {2, 16, 4, 4, 3}, {4, 2, 4, 3, 16}, {3, 0, 2, 4, 1}, tensor{0}, data_types::u8, format::bfzyx, data_types::f32, format::bfzyx
 #define CASE_PERMUTE_U8_3D_3 {1, 32, 4, 2, 1}, {1, 2, 32, 1, 4}, {4, 3, 1, 0, 2}, tensor{0}, data_types::u8, format::bfzyx, data_types::f32, format::bfzyx
 
+// permute_tile_8x8
```
Comment (yeonbok) [inline inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp:6050]: fix comment as 8x8_4x4

File: inference-engine/thirdparty/clDNN/tests/test_cases/permute_gpu_test.cpp [hunk 2]
```diff
@@ -836,3 +836,697 @@ TEST(permute_gpu_f32, basic_bfzyx_permute_0_2_3_4_1)
...
 
 }
+
+/**
+ * Test cases for permute_tile_8x8 kernel
+ *
+ * This TCs are enabled only when batch axis move to the last.
+ * i.e permute order is 0,3,1,2 or 0,4,1,2,3 or 0,5,1,2,3,4
+ */
+TEST(permute_gpu_f32_tile_8x8, normal_bfyx_0_3_1_2) {
```
Comment (yeonbok) [inline inference-engine/thirdparty/clDNN/tests/test_cases/permute_gpu_test.cpp:846]: fix names as 8x8 -> 8x8_4x4 becuase now it handles both tiling

File: inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp (hunk 1 above)
Comment (EunsooSheenIntel) [reply inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp]: Fixed the comments as 8x8_4x4

File: inference-engine/thirdparty/clDNN/tests/test_cases/permute_gpu_test.cpp (hunk 2 above)
Comment (EunsooSheenIntel) [reply inference-engine/thirdparty/clDNN/tests/test_cases/permute_gpu_test.cpp]: Fixed the comments as 8x8_4x4

**Pull request merged**